manifests/fpm.pp

```diff
@@ -45,21 +45,21 @@
 #
 class php::fpm (
   $ensure               = $::php::ensure,
-  $service_ensure       = $::php::params::fpm_service_ensure,
-  $service_enable       = $::php::params::fpm_service_enable,
-  $service_name         = $::php::params::fpm_service_name,
-  $service_provider     = undef,
```

Comment:
some parameters defaulted to `params.pp`, others to `undef` or `{}`. If it's a private class only being declared from `::php` this didn't make much sense.